#!/usr/bin/env python
PACKAGE = "cluster_assoc"

from dynamic_reconfigure.parameter_generator import *

gen = ParameterGenerator()

## gen.add("color_cost", double_t, 0, "The parameter of cost function for color", 20, 0, 100)
gen.add("dist_cost", double_t, 0, "The parameter of cost function for distance", 1.0, 0, 20)
gen.add("cost_threhold", double_t, 0, "The threshold value for the cost function used when performing cluster association", 1.5, 0, 20)
exit(gen.generate(PACKAGE, "cluster_assoc", "ClusterAssoc"))
